spec/type_aliases/cron/hour_entry_spec.rb

```diff
@@ -5,7 +5,9 @@
     it { is_expected.to allow_value( '22' ) }
     it { is_expected.to allow_value( '*' ) }
     it { is_expected.to allow_value( '*/5' ) }
+    it { is_expected.to allow_value( '2/5' ) }
```

Comment:
Why is this now allowed?  crontab allows `<range>/<skip>` and '2' is not a range